### PR TITLE
use flyteplugins with scheduler

### PR DESCRIPTION
**Pull request by vsbus**
# TL;DR
I added k8s scheduler option support to flyteplugins https://github.com/lyft/flyteplugins/commit/209c52d002b4e6a39be5d175bc1046b7e631c153

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
Importing the new flyteplugin by running
`go get github.com/lyft/flyteplugins@209c52d002b4e6a39be5d175bc1046b7e631c153`

## Tracking Issue
https://github.com/lyft/flyte/issues/<number>

## Follow-up issue
_NA_
OR
_https://github.com/lyft/flyte/issues/<number>_

### CODE REVIEW

**Comment by vsbus** [comment]
I've cut a new release: https://github.com/lyft/flyteplugins/releases

And manually updated lines in this PR by removing extra suffix from the pkg versions. I am not sure it was correct to edit manually as h1 values might be changed now. Is there is a recommended way to automatically generate this PR with the cut release?

**Comment by vsbus** [comment]
@migueltol22 please take another look

**Comment by kumare3** [comment]
@vsbus can you please link this to an issue, so in the description please fill out the linked issue number. This way when we release it helps
